content/v2.0/reference/flux/stdlib/testing/diff.md

```diff
@@ -34,16 +39,27 @@ _The `testing.diff()` function can be used to perform in-line diffs in a query._
 ## Parameters
 
 ### got
-The stream containing data to test.
+Stream containing data to test.
 _Defaults to piped-forward data (`<-`)._
 
 _**Data type:** Object_
 
 ### want
-The stream that contains the expected data to test against.
+Stream that contains the expected data to test against.
 
 _**Data type:** Object_
 
+### verbose
```

Comment:
default true?

Reply:
I'm not sure 😄... @yzhang1991 ?

Reply:
Actually I don't know about `verbose`
From the code, it does not seem to be used anywhere. I think it is dead code.

Reply:
Nm, default is `true`. Added it to the description.

Reply:
Default is true, but we did not use it in the Process() code @nathanielc

Reply:
@yzhang1991 if it's not used, should I remove it or leave it there?

Reply:
@sanderson I suggest you remove it. Setting `verbose` does not change the function behavior in any way.